src/scenes/home/team/team.js

```diff
@@ -1,31 +1,36 @@
 import React, { Component } from 'react';
```

Comment:
This component serves its purpose well. Since it was a temporary disabling of team members and an adding of another type of member to the client-side, I would've rather have edits remain out of this file, and in a new one completely.